src/3id/index.js

```diff
@@ -174,6 +175,12 @@ class ThreeId {
     return Boolean(localstorage.get(STORAGE_KEY + address.toLowerCase()))
   }
 
+  static isValid3ID(did) {
+    const parts = did.split(':')
+    if (!parts[0] === 'did' || !parts[1] === '3') return false
+    if (isIPFS.cid(did)) return true
```

Comment:
Shouldn't this line use `parts[2]` instead of `did`?
Why not just do `return isIPFS.cid(parts[2])` instead?

Reply:
fixed/resolved